scss/civicrm/contact/pages/_export.scss

```diff
@@ -62,7 +62,7 @@
 
 .CRM_Export_Form_Map {
   .help {
-    margin-bottom: 0;
+    margin-bottom: 20px;
```

Comment:
shouldn't the `.help` element have the same margin with the rest of the page no matter what the page is?

Reply:
Yes, .help should have margin-bottom: 20px; when out side form block(white shadowed area) and margin: 20px; when inside.
example,
![image](https://user-images.githubusercontent.com/26058635/36298875-51dfbbc8-1320-11e8-823d-fdb64c4f6b5e.png)

or 
![image](https://user-images.githubusercontent.com/26058635/36298909-82cd1334-1320-11e8-8005-939b74bb3e45.png)

We are using the second approach in this form

Reply:
ok so then shouldn't the selector be generic for every kind of form, instead of being specific about the export form?

Reply:
The problem doing that is,
Some of the screen makes it to look like help is outside the form block which is not true,
![image](https://user-images.githubusercontent.com/26058635/36299562-9b36fad6-1323-11e8-9f03-d8a1d6152e46.png)

Such pages are having styling done different ways (from the days when, we were keeping core templates as it is and style all things just with css without html modifications).

So, we styled padding and fonts and other stuff for .help commonly but, for margins, we rely on the style of the page already have and apply accordingly.